pkg/metrics/metrics.go

```diff
@@ -0,0 +1,26 @@
+package metrics
+
+type Metric interface {
+	Name() string
+	Value() int
```

Comment:
Let the interface enforce the description as well.

```suggestion
	Description() string
	Value() int
```

Reply:
Can't commit due to further changes. I will integrate the suggestion manually.